source/docs/api-reference/wpilib-integration/index.rst

```diff
@@ -0,0 +1,9 @@
+WPILib Integration
+==================
+
+Phoenix Pro installed in `WPILib robot projects <https://docs.wpilib.org>`__ provides extra functionality that is standard for FRC teams.
```

Comment:
"Phoenix Pro API used as part of a `WPILib robot project...` provides implementations of common WPILib interfaces that FRC teams use"